src/main/java/com/passbook/sparkeighteen/peristence/entity/UserEntity.java

```diff
@@ -0,0 +1,32 @@
+package com.passbook.sparkeighteen.peristence.entity;
+
+import lombok.*;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.*;
+import java.util.UUID;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "user",schema = "public")
```

Comment:
Why was this added please let me know.

Reply:
@Entity annotation specifies that the class is an entity and is mapped to a database table.